src/request.rs

```diff
@@ -146,7 +138,7 @@ impl DispatchSignedRequest for Client {
 
         Ok(HttpResponse {
             status: hyper_response.status,
-            body: body,
+            body: Box::new(hyper_response),
```

Comment:
The reason I changed that to a `Box` was so that we could transfer the reponse body around since it doesn't implement the `Copy` trait. The idea as I understand it is that the `Box` holds a pointer to the thing inside, and can be moved around without having to touch the underlying object.